docs/resources.md

```diff
@@ -35,7 +35,8 @@ kubetl get pods -l kritis.grafeas.io/invalidImageSecPolicy=invalidImageSecPolicy
 ## GenericAttestationPolicy CRD
 
 GenericAttestationPolicy (GAP) is a Custom Resource Definition which enforces policies based on pre-existing attestations.
-The policy expects any one attestation authority to be satisfied before allowing the container image to be admitted.
+The policy expects ALL attestation authorities to be satisfied before allowing the container image to be admitted.
+If no attestation authority is present in GAP, the container image will be admitted automatically.
```

Comment:
So this one is slightly tricky.  I think we should respect the semantics of [1] as in REQUIRE_ATTESTATION mode.  So that means that no aas is an error, rather than "admit".

[1] https://cloud.google.com/binary-authorization/docs/reference/rest/v1/Policy#AdmissionRule